<!DOCTYPE html>
<html>

<head>
    <title>SourceData Definitions</title>
    <meta charset='utf-8'>
    <script src='https://www.w3.org/Tools/respec/respec-w3c' async class='remove'></script>
    <script class='remove'>

        var respecConfig = {
            noLegacyStyle: true,
            specStatus: "unofficial",
            logos: [
                {
                    src: "http://www.embo.org/images/logos/embo_logo.jpg",
                    href: "http://embo.org",
                    alt: "embo.org",
                    id: "embo_logo"
                }
            ],
            shortName: "SourceData SOP",
            editors: [
                {
                    name: "Thomas Lemberger",
                    url: "mailto:thomas.lemberger@embo.org",
                    company: "EMBO",
                    companyURL: "http://www.embo.org"
                },
                {
                    name: "Eva Benito Garagorri",
                    url: "mailto:eva.benito@embo.org",
                    company: "EMBO",
                    companyURL: "http://www.embo.org"
                }
            ],
            previousMaturity: "2015-02-01",
            previousPublishDate: "2015-05-24",
            wg: "SourceData SOP",
            wgURI: "https://",
            wgPublicList: "https://",
            wgPatentURI: "https://",
        };

    </script>
    <style>
        html {
            background: white !important
        }

        figure {
            font-weight: bold;
        }

        figure span.fig-title {
            font-weight: normal;
        }

        figure a:visited {
            border-bottom-color: white;
        }

        figure a:hover {
            border-bottom-color: white;
        }

        /* Override for the image height and width tags included by the script */
        img {
            max-width: 100%;
            height: auto !important;
        }
    </style>
</head>

<body>
    <section id='abstract'>
        <p>
            This document describes the SourceData knowledge model used to annotated figure panels in scientific
            publications.
        </p>
    </section>

    <section id='sotd'>
        <p>
            This draft is a work in progress.
        </p>
    </section>

    
     <section>
        <h2>Introduction</h2>
        <p>Experiments in cell and molecular biology involve the empirical manipulation, observation and description of
            biological <a>entities</a>. Biological and chemical <dfn
                data-lt="entity|component|components">entities</dfn> can be entire organisms, a subset of their
            constituents or part of the experimental milieu.</p>
        <aside class="note">In this document, the terms <a>entity</a> and <a>component</a> are used interchangeably.
        </aside>
        <aside class="example">calcium, oligomycin, p53, mitochondria, liver, mus musculus, synapse, HeLa cells are
            <a>entities</a>.
        </aside>
        <aside class="example">The cell cycle, apoptosis, wound healing or type II diabetes are not entities.</aside>
        <p>SourceData description of the data presented in scientific figures specifies the <a>entities</a> that are
            relevant to the scientific meaning of the data. Annotation of attributes of such entities, of biological
            processes or diseases are not yet part of the SourceData specification described in this document.</p>
        <p>In the following sections of this document, we define the key concepts use in the SourceData annotation
            process, including the partitioning of composite figures into coherent <a>panels</a>, the <a>tagging</a> of
            <a>entities</a>, their assignments to <a>types</a> and <a>roles</a> and their <a>normalization</a> using
            external identifiers.</p>
    </section>
    <section id="conformance"></section>

    <section>
        <h2>Partitioning figures into panels</h2>
        <p>Conventional figures are composed of multiple panels and are associated with a description, the figure legend
            (or figure caption), that explains the content of the figure. While figures tend to present a heterogenous
            mixture of experimental designs and assays, individual panels are much more coherent. SourceData annotation
            is therefore carried out at the level of individual panels.</p>
        <p>A <dfn data-lt="panels">panel</dfn> SHOULD be defined as a subset of a full figure such that all of the data
            points/measurements/observations included in the panel are comparable to each other in a scientifically
            meaningful way. It is often possible to define a single common observational assay across all
            observations/data points presented in the same panel. In the majority of cases, panels correspond to the
            visual panels spontaneously delimited by authors.</p>
        <p>Each <a>panel</a> MUST be associated to its specific <dfn>panel legend</dfn>.</p>
        <aside class="note">It is often possible to generate a panel legend by including the appropriate textual
            fragments of the full figure legend. In some instances, multiple non contiguous fragments need to be spliced
            together.
        </aside>
    </section>

    <section>
        <h2>Tagging entities</h2>
        <section>
            <h2>Introduction</h2>
            <p>The primary source of information for SourceData annotation is the text of the <a>panel legend</a> and
                the image of the figure. Relevant terms from the legend or from the image are attached to a <dfn
                    data-lt="tags|tagging|tagged">tag</dfn> that specifies their <a>type</a> and <a>role</a> and that
                can be further linked to identifiers from external biological databases.</p>
        </section>

        <section>
            <h2>Tagging terms in figure legends</h2>
            <p>To be tagged, a panel MUST report experimental data. In the text of a panel legend, terms that correspond
                to specific biological and chemical <a>entities</a> SHOULD all be <a>tagged</a>.</p>
            <aside class="note">Panels that present schematics, computational simulation results, overviews, workflows
                are not tagged.
            </aside>

            <figure id='scheme'>
                <a href='img/scheme1.png' target='blank'>
                    <img src='img/scheme1.png' alt=''>
                </a>
                <figcaption>Schemes like the one shown above do not need to be annotated</figcaption>
            </figure>

            <p>In general, generic terms referring to broad classes of biological <a>components</a> (eg 'proteins',
                'cells', 'animals') SHOULD NOT be <a>tagged</a> unless they refer to the object of an assay.<p>
                    <p>Some terms such as those referring to <a>proteins</a> or <a>genes</a> can be appended with
                        prefixes or suffixes that indicate a post-translational modification, a mutation or other
                        variations of the actual base term. In such case, pre- or suffixes SHOULD be left out and only
                        the base term SHOULD be <a>tagged</a>. In other cases, a prefix is added to an entity to denote
                        a species origin, in which case the prefix SHOULD be kept.</p>
                    <aside class="example">If the cancer-related mutant form of B-RAF is mentioned as
                        <code>'B-RAF(V600E)'</code> in the text of the legend, the suffix <code>'(V600E)'</code>
                        indicating the mutation should be ignored and only <code>'B-RAF'</code> SHOULD be <a>tagged</a>.
                        Similarly if <code>'p-Akt1'</code> is designating the phosphorylated form of Akt1, only
                        <code>'Akt1'</code> SHOULD be tagged and the prefix <code>'p-'</code> SHOULD be left out.
                    </aside>
                    <aside class="example">The protein dMyc refers to the Drosophila Myc protein homolog and SHOULD be
                        tagged as <code>'dMyc'</code>.
                    </aside>
                    <aside class="note">Some components are engineered by assembling or fusing multiple sub-components,
                        which SHOULD be tagged individually. For example the term <code>'RAS-GFP'</code> referring to a
                        fusion protein between GFP and RAS should be annotated with two tags: <code>'RAS'</code> and
                        <code>'GFP'</code>.
                    </aside>
                    <aside class="note">In some instances, a fusion construct of multiple <a>entities</a> can be created
                        and referred to within the text via a shorthand symbol generated by authors. In this case,
                        <a>floating tags</a> SHOULD be created to refer to the individual components of the fusion
                        construct rather than tagging the shorthand symbol as an <a>entity</a>.
                    </aside>
        </section>

        <section>
            <h2>Adding terms missing from the figure legend</h2>
            <p>Terms can be added as <dfn data-lt="floating tag">floating tags</dfn> to complement the description of an
                experiment with <a>entities</a> that are missing from the text of the legend. These <a>entities</a>
                typically appear on the image of the figure but not in the legend.</p>
            <p>The use of <a>floating tags</a> SHOULD be restricted only to <a>entities</a> with role
                <a>controlled variable</a> or <a>assayed component</a> or <a>experimental variable</a>.If one or several of the
                3 elements are missing, they SHOULD be annoated as <a>floating tags</a>.</p>

            <figure id='floating_tag'>
                <a href='img/floating_tag.png' target='blank'>
                    <img src='img/floating_tag.png' alt=''>
                </a>
                <figcaption>Key elements missing in the figure caption should be added as a floating tag: in this
                    example, the <a>assayed component</a> AGO2 is missing and was added as a floating tag.</figcaption>
            </figure>

            <p>When a generic term is used, such as "cell" or "transcripts", to refer to a specific <a>entity</a>, ie a
                specific cell line or a specific mRNA, a <a>floating tag</a> SHOULD be used to refer explicitly to the
                specific <a>entity</a>.
            </p>
        </section>
    </section>

    <section>
        <h2>Entity types & normalization</h2>

        <section>
            <h2>Types</h2>
            <p><a>Entities</a> are assigned to one of seven <dfn data-lt="type">types</dfn> spanning successive levels
                of biological organisations (<a>Table 1</a>). Each type is mutually exclusive. If an <a>entity</a> is
                linked to an identifier from an external resource, it SHOULD use the resource associated to its type
                according to <a>Table 1</a>:</p>

            <table>
                <caption><dfn>Table 1</dfn>. Entity types and the corresponding resources to which they can be linked.
                </caption>
                <tr>
                    <th>Entity Type</th>
                    <th>Primary Resource</th>
                    <th>Secondary Resource</th>
                <tr>
                    <td><dfn data-lt="small molecule">Small molecules</dfn>
                    <td><a href="http://www.ebi.ac.uk/chebi/">ChEBI</a>
                    <td><a href="http://www.ncbi.nlm.nih.gov/">PubChem</a>
                <tr>
                    <td><dfn data-lt="gene">Genes</dfn></td>
                    <td><a href="http://www.ncbi.nlm.nih.gov/gene/">NCBI Gene</a></td>
                    <td><a href="http://rfam.xfam.org">Rfam</a>
                <tr>
                    <td><dfn data-lt="protein">Proteins</dfn>
                    <td><a href="http://uniprot.org">UniprotKB/Swiss-Prot</a>
                    <td>
                <tr>
                    <td><dfn data-lt="cellular component">Cellular components</dfn>
                    <td><a href="http://geneontology.org">Gene Ontology (Cellular Components)</a>
                    <td>
                <tr>
                    <td><dfn data-lt="cell line|cell lines|cell type|cell types">Cell types and cell lines</dfn>
                    <td><a href="http://web.expasy.org/cellosaurus/">Cellosaurus</a>
                    <td><a href="http://obofoundry.org/ontology/cl.html">Cell Ontology</a>
                <tr>
                    <td><dfn data-lt="tissue|tissues|organ|organs">Tissues & organs</dfn>
                    <td><a href="http://uberon.org">Uberon</a>
                    <td>
                </tr>
                <tr>
                    <td><dfn data-lt="organism|organisms|species">Organisms & species</dfn></td>
                    <td><a href="http://www.ncbi.nlm.nih.gov/taxonomy/">NCBI Taxonomy</a>
                    <td>
                </tr>
            </table>

            <aside class="example"> ATP is a <a>small molecule</a>, creb1 is a <a>gene</a>, CREB1 is a <a>protein</a>,
                the Golgi apparatus is a <a>cellular component</a>, HEK293 cells is a <a>cell line</a>, the retina is a
                <a>tissue</a> and Saccharomyces cerevisiae and PhiX174 are <a>organisms</a>.</aside>
            <p>If an <a>entity</a> does not fit any of the predefined <a>types</a>, the <dfn>undefined</dfn> <a>type</a>
                is assigned. By definition, <a>undefined</a> <a>entities</a> cannot be linked to any external resources.
            </p>
            <aside class="note"><a>Undefined</a> <a>entities</a> are <a>tagged</a> as such to enable retrospective
                analysis of whether additional resources and types should be added in a subsequent version of the
                SourceData model.</aside>
            <p>In omics experiments, the number of entities measured is too large to be all listed explicitly. It is
                then nevertheless possible to represent the experiments by adding as <a>floating tag</a> the reserved
                words <code>multiple components</code> and assign the appropriate <a>type</a> (see also Appendix <a
                    href="#representing-omics-experiments"></a>).</p>
        </section>

        <section>
            <h2>Linking to standard identifiers (normalization)</h2>
            <p>In the <dfn data-lt="normalize|normalized|normalizes|normalizing">normalization</dfn> process,
                <a>entities</a> SHOULD be linked to one or several identifiers of the external resources corresponding
                to the entity's <a>type</a> (see <a>Table 1</a>). If an <a>entity</a> is linked to multiple identifiers,
                it MUST mean that there is uncertainty about the exact identity of the <a>entity</a>.</p>
            <aside class="example">If the term <code>'Akt'</code> is used to refer to the mouse protein Akt, it is
                unclear whether it refers to the Akt1, Akt2 or Akt3 isoform. As such, the term will be <a>normalized</a>
                to the external identifiers <code>Uniprot:P31750; Uniprot:Q60823; Uniprot:Q9WUA6</code> corresponding to
                Akt1, Akt2 and Akt3, respectively. </aside>
            <aside class="note">In the case of <a>entities</a> that are <a>normalized</a> to identifiers from ontologies
                and taxonomies (<a>cellular components</a>, <a>cell types</a>, <a>tissues</a> and <a>organisms</a>),
                uncertainty about the identity of the <a>entity</a> SHOULD be expressed by <a>normalizing</a> it to a
                sufficiently generic concept in the ontology/taxonomy. For example, the strain <code>'HSV-1 (F)'</code>
                does not have a specific entry in the NCBI Taxonomy database but can be normalised to the more generic
                taxon <code>'HSV-1'[NCBI Taxonomy:10304]</code>.</aside>

            <p>Linking <a>reporter components</a> or <a>normalizing components</a> to an external identifier is
                OPTIONAL.</p>

            <p>Identifiers pointing to curated records of external databases SHOULD be preferred over identifiers
                referring to non-curated records. If relevant records exist both in the primary and secondary resources
                listed in <a>Table 1</a>, identifiers from the primary resource SHOULD be used.</p>

        </section>
    </section>

    <section>
        <h2>Entity roles</h2>
        <section>
            <h2>Introduction</h2>
            <p>Biological <a>components</a> listed in the caption of a figure each play a different <dfn
                    data-lt="roles">role</dfn> in the experimental design: some <a>components</a> are altered in a
                controlled manner, others remain untouched by the experimenter, and some are directly or indirectly
                assayed to perform measurements or observations. Accordingly, the following <a>roles</a> are defined:
            </p>
            <ul>
                <li><a>Biological component</a></li>
                <li><a>Assayed component</a></li>
                <li><a>Controlled variable</a></li>
                <li><a>Reporter component</a></li>
                <li><a>Normalizing component</a></li>
                <li><a>Experimental variable</a></li>
            </ul>

            <aside class="note">For all types of entities, if there are multiple instances of the same entity in the
                figure legend, all instances of the tag should be captured.
            </aside>
        </section>

        <section>
            <h2>Biological components</h2>
            A <dfn data-lt="biological components">biological component</dfn> is a generic category for any
            experimentally relevant <a>component</a> which does not fit any of the other defined roles within
            SourceData. Often it will contain the organism, the cell, or a generic treatment that is present across all
            conditions.
        </section>

        <section>
            <h2>Assayed components</h2>
            <p>An <dfn data-lt="assayed components|assayed">assayed component</dfn> is the <a>component</a> that is
                measured or observed.</p>
            <aside class="example">The proteins detected on a Western blot are the <a>assayed components</a> with the
                exception of the loading control, if any, which is a <a>normalizing component</a> (see below).</aside>

            <figure id='normalizing_western'>
                <a href='img/normalizing_western.png' target='blank'>
                    <img src='img/normalizing_western.png' alt=''>
                </a>
                <figcaption>In the example above, tubulin is the normalizing component.</figcaption>
            </figure>

            <aside class="note">If a molecular marker, for example the protein EEA1, is assayed to visualize a higher
                order structure, for example endosomal vesicles, the marker (EEA1 in this example) is tagged as an
                <a>assayed component</a>. The higher order structure (endosomal vesicles in this example) is
                <a>tagged</a> as <a>assayed component</a> only if it is explicitly highlighted on the image or a
                property of the entity (such as number/localisation) is mentioned in the text of the legend.</aside>

            <figure id='molecular marker'>
                <a href='img/markers.png' target='blank'>
                    <img src='img/markers.png' alt=''>
                </a>
                <figcaption>In this case, HDEL is an ER marker, so it is captured as an <a>assayed component</a>. In
                    addition, because there is a specific reference to the ER in the legend, ER is also captured as an
                    <a>assayed component</a>. Note that in this example there is a second marker, CMAC, which is however
                    a reporter as is therefore captured as a <a>reporter</a> for vacuole, which is labelled as an
                    <a>assayed component</a> </figcaption>
            </figure>
        </section>

        <section>
            <h2>Controlled variables</h2>
            <p>An <dfn data-lt="controlled variables|controlled variable|perturbation|perturbations">controlled variable (also called perturbation, intervention, manipulation, alteration or independent variable)</dfn> is a <a>component</a> that is experimentally altered. An <a>controlled variable</a>
                MUST be targeted and MUST be controlled. This implies that the experiment MUST involve the same
                experimental system across experimental groups and MUST involve a comparison between several
                experimental groups to test whether the controlled variable causes an effect on the <a>assayed component</a>.
            </p>
            <aside class="example">The function of the gene creb1 can be investigated by comparing creb1 wt (control
                group) to creb1-/- knockout (test group) mice; in this experiment, creb1 is the <a>controlled variable</a>. If,
                and only if, it is appropriately controlled, the purpose of such an experiment is to infer a
                cause-and-effect relationship, whether direct or indirect, between the <a>controlled variable</a> and the
                <a>assayed component</a>.</aside>
            <aside class="warning">
                <p>If a drug (cycloheximide, for example) is applied across <b>ALL</b> experimental groups, it is
                    <b>NOT</b> considered as an <a>controlled variable</a>, since there is no control group to compare the
                    effect of the drug across conditions. An <a>controlled variable</a> MUST be controlled. Accordingly, in
                    such context, the drug SHOULD be tagged as <a>biological component</a>.</p>
                <p>Similarly, if a cell strain harboring the same genetic mutation is used across <b>ALL</b>
                    experimental groups, the mutated gene is <b>NOT</b> an <a>controlled variable</a> but a generic
                    <a>biological component</a>.</p>
            </aside>


            <figure id='biological component'>
                <a href='img/biological_component.png' target='blank'>
                    <img src='img/biological_component.png' alt=''>
                </a>
                <figcaption>MOB1A and LATS2 are both overexpressed across all conditions in this experiment, i.e., they
                    are not controlled for. The only entity that is differentially manipulated in this experiment is
                    TRIP6. Thus, both MOB1A and LATS2 are considered <a>biological components</a> and TRIP6 is cosidered
                    the <a>controlled variable</a></figcaption>
            </figure>




            <aside class="note">The target of an experimental manipulation is usually tagged as <a>controlled variable</a>.
                <a>Small molecules</a> such as drugs, inhibitors, agonists and other pharmacological compounds are
                usually considered as the <a>controlled variable</a> when their effects are compared across experimental
                groups. An exception is when a <a>small molecule</a> (for example, doxycycline, IPTG, arabinose) is used
                to manipulate the activity of an engineered circuit controlling the actual entity of interest (for
                example, a gene whose expression needs to be varied), in which case the <a>entity</a> of interest is
                considered as the actual <a>controlled variable</a> and the triggering compound (doxycycline, IPTG, arabinose)
                is captured as <a>biological component</a>.</aside>
            <aside class="example">If cells are treated with different doses of the PKA inhibitor H89, H89 is tagged as
                the <a>controlled variable</a>. </aside>
            <aside class="note">In experiments that test the action of an entity over time, the entity is tagged as an
                controlled variable only if a control group is tested or if the the time=0 is also shown as point of
                comparison.</aside>
            <aside class="example">In a siRNA-mediated knock down experiment, the gene targeted by the siRNA is tagged
                as an <a>controlled variable</a>.</aside>
            <aside class="note">An <a>controlled variable</a> MUST involve controlled experimental conditions. It is therefore
                common that control experimental groups are treated with a neutral compound, for example the solvent
                used to dissolve the administered drug. By convention, such <a>components</a> MUST be assigned the
                generic role <a>biological component</a>.</aside>
            <aside class="note">In transfection experiments for overexpression, the main entity of the construct used
                for transfection should be labelled as <a>controlled variable</a> of type gene and if detected, the protein
                should be tagged as <a>assayed component</a> of type protein.</aside>


            <figure id='transfection type gene'>
                <a href='img/controlled variable_transfection_type_gene.png' target='blank'>
                    <img src='img/controlled variable_transfection_type_gene.png' alt=''>
                </a>
                <figcaption>Note that the controlled variable through transfection of FLAG-TRIP6 is of type gene. This example
                    is also a good example of where it is appropriate to add a <a>floating tag</a> when key elements
                    (<a>assayed</a> or <a>controlled variable</a>) are missing from the figure legend.</figcaption>
            </figure>
        </section>

        <section>
            <h2>Reporter components</h2>
            <p>A <dfn data-lt="reporter components|reporter|reporters">reporter component</dfn> is used as a proxy to
                measure or observe indirectly an <a>assayed component</a> of interest to which it is linked as part of a
                synthetic or engineered construct.
                <aside class="example">A RAS-GFP fusion protein includes the RAS protein as an <a>assayed component</a>
                    and GFP as a <a>reporter component</a>.</aside>
                <aside class="example">The luciferase gene can be used a <a>reporter</a> gene to monitor the
                    transcriptional activity of a given gene promoter, which is the actual <a>assayed component</a> of
                    interest.</aside>


                <figure id='luciferase'>
                    <a href='img/luciferase.png' target='blank'>
                        <img src='img/luciferase.png' alt=''>
                    </a>
                    <figcaption>In this luciferase experiment, what is being measured is the effect of miR-16-5p on the
                        3'UTR of Bdnf, so Bdnf is labelled as the assayed component.</figcaption>
                </figure>




                <p>Linking <a>reporter components</a> or <a>normalizing components</a> to an external identifier is
                    OPTIONAL.</p>

        </section>

        <section>
            <h2>Normalizing components</h2>
            <p>A <dfn data-lt="normalizing components">normalizing component</dfn> is a <a>component</a> that is assayed
                for the purpose of providing baseline measurements from each experimental group so that the the data can
                be normalized across groups.</p>
            <aside class="example">The proteins beta-actin or GAPDH are often assayed to serve as loading control in
                Western blots and are then <a>tagged</a> as <a>normalizing component</a>.</aside>
            <p>Linking <a>normalizing components</a> to an external identifier is OPTIONAL.</p>
        </section>

        <section>
            <h2>Experimental variables</h2>
            <p>When a <a>component</a> is used to compare multiple experimental groups but it is not possible to clearly
                infer a cause-and-effect relationship between this <a>component</a> and the <a>assayed components</a> of
                the experiment, the <a>component</a> is said to be an <dfn data-lt="experimental variables">experimental
                    variable</dfn>.</p>
            <aside class="example">If the expression of a given gene is measured across tissues and cell lines,
                including liver, muscle, brain, HEK293 and HeLa cells, the tissues or cell types are tagged as
                <a>experimental variables</a>.</aside>

            <figure id='experimental variable'>
                <a href='img/experimental_variable.png' target='blank'>
                    <img src='img/experimental_variable.png' alt=''>
                </a>
                <figcaption>In this example, expression of ArfGAP1 is assayed in several cell types in parallel. This is
                    the prototypical definition of an <a>experimental variable</a>.</figcaption>
            </figure>


        </section>
    </section>

    <section class="appendix">
        <h2>Tagging experimental assays</h2>
        <p>In addition to entities, SourceData is also experimenting with the tagging of non-entity terms. In
            particular, the <dfn data-lt="experimental assay|experimental assays">experimental assay</dfn> used to
            observed or measure the <a>assayed components</a> of an experiment are tagged and normalized to identifiers
            either from the BioAssay ontology (<a href="bioassayontology.org/">BAO</a>) or from the Ontology for
            Biomedical Investigations (<a href="http://obi-ontology.org/">OBI</a>)</p>
        <aside class="note">At present, SourceData only captures experimental assays used to actually collect data,
            i.e., associated with the <a>assayed component</a>. The experimental assay used to induce the
            <a>controlled variable</a> should not be tagged. In addition, SourceData strives to capture the <a>experimental
                assay</a> itself and not necessarily the METHOD. If both the assay and the method are explicitly
            mentioned in the figure legend, SourceData captures both. If not, only the <a>experimental assay</a> is
            captured, either as tag in the figure legend or as a floating tag if missing.</aside>
        <aside class="example">If a figure represents images from an immunostaining, it suffices to annotate
            "immunostaining" as the <a>experimental assay</a> and there is no need to add a floating tag for
            "microscopy" if this is not present in the figure legend</a>.</aside>

        <figure id='experimental assay vs method'>
            <a href='img/assay_method.png' target='_blank'>
                <img src='img/assay_method.png' alt=''>
            </a>
            <figcaption>In this example, the <a>experimental assay</a> is 'gene expression measurement' and the method
                is qRT-PCR. Because qRT-PCR is mentioned in the figure legend, it is captured. However, 'gene
                expression' is also added as a <a>floating tag</a> to capture the <a>experimental assay</a> for the
                experiment.</figcaption>
        </figure>
    </section>


    <section class="appendix">
        <h2><dfn data-lt="time-related variable">Time-related variables</dfn></h2>
        <p><a>Time-related variable</a>: add a floating tag “time course” or “age”
            to indicate comparison of an controlled variable or an assayed component over time within an experiment.</p>
    </section>


    <section class="appendix">
        <h2><dfn data-lt="physical variable">Physical variables</dfn></h2>
        <p>A <a>Physical variable</a> refers to particular experimental conditions,
            e.g. 'cold exposure', 'footshock', etc. Add them only when explicitly mentioned in the figure legend.
        </p>
    </section>


    <section class="appendix">
        <h2>Special cases: Protein complexes, FACS, cell cycle phases and DNA staining</h2>
        <p>Some experimental designs are unique in nature. For the following cases, these guidelines should be observed:
            <ul>
                <li>Protein complexes: although the intuitive normalization for protein complexes would be of type
                    protein, it is more adequate to assign them the type "subcellular component" because the Gene
                    Ontology (GO) database contains normalized references for protein complexes. An example of this
                    would be RNA Polymerase II, which is made up of a number of individual subunits. See the example
                    below</li>
                <figure id='protein complexes'>
                    <a href='img/protein_complexes.png' target='_blank'>
                        <img src='img/protein_complexes.png' alt=''>
                    </a>
                    <figcaption> Notice how in this example RNA-Pol-II is assigned the type "subcellular component",
                        which can be normalize to the GO database.</figcaption>
                </figure>
                <li>FACS experiments: for FACS experiments, SourceData captures both the cells and the DNA (or whichever
                    element is stained and sorted) as <a>assayed components</a></li>
                <figure id='FACS'>
                    <a href='img/FACS.png' target='_blank'>
                        <img src='img/FACS.png' alt=''>
                    </a>
                    <figcaption>In this example, HeLa cells are stained with AnnexinV to measure cell death. Both HeLa
                        cells and AnnexinV are captured as <a>assayed components</a>.</figcaption>
                </figure>
                <li>Cell cycle phases: at present, SourceData does not include cell cycle phases as time elements. Cell
                    cycle phases may be added when appropriate to <a>experimental variable</a> or <a>biological
                        component</a> as class 'unknown'</li>
                <li>DNA staining: for DNA stains like BrdU, EdU, etc., SourceData captures DNA as the <a>assayed
                        component</a> and 'BrdU staining' as the <a>experimental assay</a></li>
            </ul>
        </p>
    </section>

    <section class="appendix">
        <h2>Representing 'omics' experiments</h2>
        <p>For experiments performing a large number of measurements (>15-20), for example in metabolomics, genomics,
            transcriptomics and proteomics, the assayed components cannot be listed individually. The following
            tentative guidelines are then followed:
            <ul>
                <li>The reserved expression <code>'multiple components'</code> should be included as floating tag, with
                    the relevant entity <a>type</a> and with the role <a>assayed component</a>.</li>
                <figure id='multiple components'>
                    <a href='img/multiple_components3.png' target='_blank'>
                        <img src='img/multiple_components3.png' alt=''>
                    </a>
                    <figcaption>Note how individual metabolites are captured under the expression
                        <code>multiple components</code> in this example.</figcaption>
                </figure>
                <li>Both the measurement type (protein expression, protein-DNA interactions, protein-protein
                    interactions, etc...) and the assay technology (experimental platform such as the sequencing
                    platform, mass spec platform, etc...) as mentioned in Materials & Methods should be tagged as
                    <a>experimental assay</a>, if necessary as <a>floating tag</a>.</li>
                <li>The experimental system that is profiled should be added as <a>biological component</a>, even if it
                    requires adding a <a>floating tag</a>.</li>
                <aside class="example">In proteomics, if mass spectrometry is used to measure protein-protein
                    interactions in HeLa cell extracts, for example, the terms <code>'mass spectrometry'</code> (<a
                        href="http://identifiers.org/bao/0000055"><code>BAO:0000055</code></a>) and
                    <code>'protein-protein interaction assay'</code> (<a
                        href="http://identifiers.org/bao/0002990"><code>BAO:0002990</code></a>) will both be added as
                    <a>experimental assay</a> <a>floating tags</a>, as well as <code>'HeLa'</code> (<a
                        href="http://web.expasy.org/cellosaurus/CVCL_0030"><code>CL/CVCL:CVCL_0030</code></a>) as
                    general <a>biological component</a> that specifies the experimental system.</aside>

                <li>For panels representing different analyses of the same underlying raw data (e.g. heatmap, PCA,
                    correlation matrix), only the <b>primary representation</b>, i.e., first instance of the data should
                    be tagged.</li>
                <aside class="example">In a microarray experiments, the <a>assayed component</a> will be
                    <code>'multiple components'</code> of class 'gene'. If for example the heatmap is the primary
                    representation of the data, then the floating tag will be associated with that panel. If the next
                    panel is for example a PCA plot, then this is considered the second instance of the same primary
                    data and it SHOULD not be tagged.</aside>
            </ul>
    </section>

    <section class="appendix">
        <h2>Guidelines for author queries</h2>
        <p> When in doubt about the normalization of an entity, authors can be queried via the validation interface. The following should be systematically queried to ensure accurate annotation: </p>
            
        <h4>Systematic queries</h4>    
        <p>Authors should always be asked to <strong>confirm the assigned id</strong> for the following entities:</p>
        <ul>
            <li>Disease</li>
            <li>Cell lines (particular attention to hierarchy)</li>
            <li>Strains</li>
            <li>Cell types (particular attention to immune lineages)</li>
            <li>Molecules (when normalized to PubChem)</li>
        </ul>
        <br>
        <p>In addition, queries should be raised in these scenarios <strong>asking authors to provide or clarify an id:</strong></p>
        <h4>Context-dependent queries</h4>
        <ul>
            <li>Proteins:
                <ul>
                    <li>Several id's</li>
                    <li>No id</li>
                    <li>Species of protein unclear</li>
                </ul>
            </li>
            <li>Genes:
                    <ul>
                        <li>Several id's</li>
                        <li>No id</li>
                        <li>Species of gene unclear</li>
                    </ul>
                </li>
        </ul>
        <p>The full list and content of pre-compiled author queries can be found <a href='https://docs.google.com/document/d/1HIhh0g4u_FIO8MAOdwsh18fj5uljcct6AbvKniQ5r1E/edit?usp=sharing'>here</a> </p>
    </section>
</body>
</html>